src/corenfc.cs

```diff
@@ -144,9 +152,141 @@ interface NFCIso15693Tag : NFCTag {
+#if XAMCORE_4_0
+		[Abstract]
+#endif
+		[Export ("selectWithRequestFlags:completionHandler:")]
+		void Select (RequestFlag flags, Action<NSError> completionHandler);
+
+		[iOS (13,0)]
+#if XAMCORE_4_0
+		[Abstract]
+#endif
+		[Export ("resetToReadyWithRequestFlags:completionHandler:")]
+		void ResetToReady (RequestFlag flags, Action<NSError> completionHandler);
+
+		[iOS (13,0)]
+#if XAMCORE_4_0
+		[Abstract]
+#endif
+		[Export ("writeAFIWithRequestFlag:afi:completionHandler:")]
+		void WriteAfi (RequestFlag flags, byte afi, Action<NSError> completionHandler);
+
+		[iOS (13,0)]
+#if XAMCORE_4_0
+		[Abstract]
+#endif
+		[Export ("lockAFIWithRequestFlag:completionHandler:")]
+		void LockAfi (RequestFlag flags, Action<NSError> completionHandler);
+
+		[iOS (13,0)]
+#if XAMCORE_4_0
+		[Abstract]
+#endif
+		[Export ("writeDSFIDWithRequestFlag:dsfid:completionHandler:")]
+		void WriteDsfi (RequestFlag flags, byte dsfid, Action<NSError> completionHandler);
+
+		[iOS (13,0)]
+#if XAMCORE_4_0
+		[Abstract]
+#endif
+		[Export ("lockDFSIDWithRequestFlag:completionHandler:")]
+		void LockDfsi (RequestFlag flags, Action<NSError> completionHandler);
+
+		[iOS (13,0)]
+#if XAMCORE_4_0
+		[Abstract]
+#endif
+		[Export ("getSystemInfoWithRequestFlag:completionHandler:")]
+		void GetSystemInfo (RequestFlag flags, NFCGetSystemInfoCompletionHandler completionHandler);
+
+		[iOS (13,0)]
+#if XAMCORE_4_0
+		[Abstract]
+#endif
+		[Export ("getMultipleBlockSecurityStatusWithRequestFlag:blockRange:completionHandler:")]
+		void GetMultipleBlockSecurityStatus (RequestFlag flags, NSRange blockRange, Action<NSNumber[], NSError> completionHandler);
+
+		[iOS (13,0)]
+#if XAMCORE_4_0
+		[Abstract]
+#endif
+		[Export ("customCommandWithRequestFlag:customCommandCode:customRequestParameters:completionHandler:")]
+		void CustomCommand (RequestFlag flags, nint customCommandCode, NSData customRequestParameters, Action<NSData, NSError> completionHandler);
+
+		[iOS (13,0)]
+#if XAMCORE_4_0
+		[Abstract]
+#endif
+		[Export ("extendedReadSingleBlockWithRequestFlags:blockNumber:completionHandler:")]
+		void ExtendedReadSingleBlock (RequestFlag flags, nint blockNumber, Action<NSData, NSError> completionHandler);
+
+		[iOS (13,0)]
+#if XAMCORE_4_0
+		[Abstract]
+#endif
+		[Export ("extendedWriteSingleBlockWithRequestFlags:blockNumber:dataBlock:completionHandler:")]
+		void ExtendedWriteSingleBlock (RequestFlag flags, nint blockNumber, NSData dataBlock, Action<NSError> completionHandler);
+
+		[iOS (13,0)]
+#if XAMCORE_4_0
+		[Abstract]
+#endif
+		[Export ("extendedLockBlockWithRequestFlags:blockNumber:completionHandler:")]
+		void ExtendedLockBlock (RequestFlag flags, nint blockNumber, Action<NSError> completionHandler);
+
+		[iOS (13,0)]
+#if XAMCORE_4_0
+		[Abstract]
+#endif
+		[Export ("extendedReadMultipleBlocksWithRequestFlags:blockRange:completionHandler:")]
+		void ExtendedReadMultipleBlocks (RequestFlag flags, NSRange blockRange, Action<NSData [], NSError> completionHandler);
```

Comment:
All the above have handler, do we need to use Async?

Reply:
nope